script/sem-ver-action.py

```diff
@@ -0,0 +1,70 @@
+#  imports
+import os
+import git
+from git import Repo
+
+#  variables
+script_path = os.getcwd()
+remote_url = "git@github.com:shiri2001/training.git"
+branch = "main"  # noqa:E501 change later to main so it only runs on main
+repo = Repo(script_path)
```

Comment:
The General Variables section is a good idea. It gives your reviewer/maintainer a single place  adding additional features, or fixing the code to make any necessary changes (such as variable updates).
Everything make sense except the `repo` why you declared it here?